### PR TITLE
Add Outputs to just the table for now

### PR DESCRIPTION
**Pull request by opp-svega**
- Add an outputs summary to the table as a separate render
- Ran `go fmt` on all the files
- Added a new flag to disable outputs from view

### CODE REVIEW

File: terraform_state/.terraform_state.go.swp
Comment (dineshba) [inline terraform_state/.terraform_state.go.swp:1]: Looks this file should not be added to git

File: writer/table.go [hunk 1]
```diff
@@ -39,12 +41,43 @@ func (t TableWriter) Write(writer io.Writer) error {
 
 	table.Render()
 
+	// Disable the Output Summary if there are no outputs to display
+	if t.outputs == true {
```
Comment (dineshba) [inline writer/table.go:45]: Instead of adding new outputs flag, `if len(t.output_changes) > 0` shall we display outputs table ?

**Review by dineshba — COMMENTED** [review]
Thanks a lot for this PR.

File: terraform_state/.terraform_state.go.swp
Comment (opp-svega) [reply terraform_state/.terraform_state.go.swp:1]: Ill remove this file, must have been added by accident.

**Review by dineshba — APPROVED** [review]

**Comment by dineshba** [comment]
@opp-svega Thanks for the update.

**Comment by dineshba** [comment]
Could you be interested in supporting outputs in other writers ? @opp-svega

**Comment by opp-svega** [comment]
@dineshba Ill make some time this week to try to update the rest.

**Comment by Terraformadillo** [comment]
@dineshba @opp-svega This could be very dumb question but asking anyway. What is use case for the Change/Output section? Its usally always blank for me (example below).

|  CHANGE  |                                                           RESOURCE                                                           |
|----------|------------------------------------------------------------------------------------------------------------------------------|
| recreate | `module.client_infrastructure.module.batch_database_restore_job_definition[0].aws_batch_job_definition.batch_job_definition` |
| update   | `module.client_infrastructure.module.api_zone_record[0].aws_route53_record.record`                                           |
|          | `module.client_infrastructure.module.ecs_autoscaling_target_web[0].aws_appautoscaling_target.appautoscaling_target`          |
|          | `module.client_infrastructure.module.ecs_autoscaling_target_worker[0].aws_appautoscaling_target.appautoscaling_target`       |
|          | `module.client_infrastructure.module.ecs_task_service_beat[0].aws_ecs_service.ecs_service`                                   |
|          | `module.client_infrastructure.module.ecs_task_service_web[0].aws_ecs_service.ecs_service`                                    |
|          | `module.client_infrastructure.module.ecs_task_service_worker[0].aws_ecs_service.ecs_service`                                 |

| CHANGE | OUTPUT |
|--------|--------|

**Comment by opp-svega** [comment]
@Terraformadillo That section is only populated when there are changes to the Terraform Outputs. If you dont have terraform outputs, or the values of the outputs dont change, then this section will be blank.

**Comment by Terraformadillo** [comment]
@opp-svega thanks for explaination

**Comment by dineshba** [comment]
If there are no output changes, Shall we not show table itself ? Thoughts ? @opp-svega

**Comment by opp-svega** [comment]
@dineshba That sounds good to me, ill add the logic to hide the outputs.

**Comment by opp-svega** [comment]
@dineshba Please see https://github.com/dineshba/tf-summarize/pull/41

**Pull request merged**